### PR TITLE
Student profile page has descriptions now

### PR DESCRIPTION
**Pull request by Sher1nShaj1**

### CODE REVIEW

File: app/src/main/res/layout/activity_orgs.xml [hunk 1]
```diff
@@ -66,8 +64,22 @@
...
+                android:layout_height="wrap_content"
+                android:layout_below="@+id/tv_name"
+                android:layout_centerHorizontal="true"
+                android:layout_marginTop="15sp"
+                android:fontFamily="sans-serif-light"
+                android:text="Description"
+                android:textAllCaps="true"
+                android:textColor="@color/white"
+                android:textSize="20sp"
+                android:textStyle="bold" />
```
Comment (zalexchen) [inline app/src/main/res/layout/activity_orgs.xml:82]: Might want to refactor the id to tv_description, but other than that, LGTM, I'll implement with this immediately

Comment (ChloeMJM) [reply app/src/main/res/layout/activity_orgs.xml:82]: lgtm. can't see any conflicts.

**Comment by rahulsheth1** [comment]
All changes look good!

File: app/src/main/res/layout/activity_orgs.xml (hunk 1 above)
Comment (samiachowdhury) [reply app/src/main/res/layout/activity_orgs.xml:82]: Looks good, I didn't get any errors.

**Pull request merged**